### DISCUSSION TITLE
End User Defined/Created Dynamic Forms

### DISCUSSION THREAD

**Discussion by KeanuTang** [filamentphp/filament#12394] [Help]
### Package

Form builder

### Package Version

v3.2

### How can we help you?

Does anyone have experience using Filament Forms to support user-defined/user-created forms? Did you have to build your own UI to create and edit the forms?

Here's the context. We run a Laravel application where our customers can create internship application forms. We're looking to upgrade our form builder, and we came across Filament. Our customers need to be able to edit and manage their internship application forms, so these cannot be hardcoded in PHP code.

I'm not opposed to having to create the builder UI, but it would be nice to not reinvent the wheel if this kind of thing has been done before.

**Comment by dissto** [comment]
I suppose [Builder](https://filamentphp.com/docs/3.x/forms/fields/builder) is the right choice for the job. 😋

**Reply by juanfernandojfc** [reply]
but how can i save the form schema, and load it into another class, like having a select component then choosing a form schema and load it into the actual form

**Reply by haywardTsoi** [reply]
it will be json
